template/{{.input_root_dir}}/.azure/devops-pipelines/{{.input_project_name}}-bundle-cicd.yml.tmpl

```diff
@@ -49,9 +49,9 @@ stages:
 
     # Validate bundle to be deployed to the staging workspace
     - script: |
-        databricks bundle validate -e staging  
+        databricks bundle validate -t staging
       workingDirectory: {{template `project_name_alphanumeric_underscore` .}}
-      displayName: 'Validate bundle for staging environment'
+      displayName: 'Validate bundle for staging target(environment)'
```

Comment:
You could also omit the suffix, and have just "Validate bundle for staging".

I recommend getting rid of mentions of "environment".

Reply:
Sounds good. I've removed the `environments` and updated docs. Could you take another look? Thank you.